sandbox/apparmor/apparmor.go

```diff
@@ -437,3 +502,11 @@ func MockFeatures(kernelFeatures []string, kernelError error, parserFeatures []s
 	}
 
 }
+
+func MockSnapdAppArmorSupportsReexec(new func() bool) (restore func()) {
+	old := snapdAppArmorSupportsReexec
+	snapdAppArmorSupportsReexec = new
```

Comment:
you can use `testutils.Backup()`

Reply:
Sorry, I'm afraid I gave you a wrong suggestion: we cannot use `testutils` in snapd, only in the tests; can this function be moved to `export_test.go`?

If not, then I'm afraid you should revert it to your original implementation.

Reply:
No worries - done in  76a8d13428